### ISSUE TITLE
(code) GetLatestVersion doesn't work in CLI

### ISSUE THREAD

**Issue by yevgenypats** [cloudquery/cloudquery#2076]
Follow-up on this PR where I commented the test, https://github.com/cloudquery/cloudquery/pull/2074/files 

Seems get latest CLI version stopped working. worth adding this back.

**Comment by hermanschaaf** [comment]
This isn't working because the version file is currently return an empty version:

```
$ curl https://versions.cloudquery.io/v2/cli.json
{ "latest": "" }
```

And it looks like it's deliberate while we're in pre-release: https://github.com/cloudquery/cloudquery/blob/main/.github/workflows/release_cli.yml#L70-L72

I'm trying to understand whether it would be okay to update that file right now, and what the effects will be 🤔

**Comment by erezrokah** [comment]
Yes the version files only get updated for official releases.

But I think it's ok to update those manually since v0 uses `v1` versions site and `v1` (pre-release) uses `v2` versions site.
I created the `v2` versions site per https://github.com/cloudquery/cloudquery/pull/1957 as to not breaking existing versions on the CLi.

TLDR: If we change the versions in `v2` nothing should breaking as the only consumer is the v1 pre-release

**Comment by erezrokah** [comment]
BTW for the new destination plugins, as those were officially releases, the versions are there:
https://github.com/cloudquery/cloudquery/blob/f1a1caa1ac89e38cfb289e15bddf60a4d7710234/sites/versions/v2/destination-postgresql.json#L1
https://github.com/cloudquery/cloudquery/blob/f1a1caa1ac89e38cfb289e15bddf60a4d7710234/sites/versions/v2/destination-postgresql.json#L1

**Comment by erezrokah** [comment]
And if this PR was mostly for the CLI, I think the only consumer of the CLI JSON file is our GitHub action:
https://github.com/cloudquery/setup-cloudquery/blob/1c8abd16eb48f102853bbd8863f84cf2750a60a9/src/main.ts#L16

**Comment by yevgenypats** [comment]
I think this is solved now.
